examples/src/Modalizer.stories.tsx

```diff
@@ -1,11 +1,26 @@
 import * as React from 'react';
-import { getAbilityHelpersAttribute } from 'ability-helpers';
+import { Meta } from '@storybook/react';
+import { getAbilityHelpersAttribute, getCurrentAbilityHelpers, getDeloser, getModalizer } from 'ability-helpers';
 import { Modal } from './components/Modal';
 
 // eslint-disable-next-line import/no-anonymous-default-export
 export default {
-  title: 'Modalizer',
-};
+    title: 'Modalizer',
+    decorators: [
+        (Story) => {
+            const ah = getCurrentAbilityHelpers(window);
+            
+            // These APIs need to initialized before use, no reason to do it for all stories yet
+            if (ah) {
+                getModalizer(ah);
+                getDeloser(ah);
+            }
```

Comment:
@mshoho  Should all APIs be initialized in examples by default ?

Reply:
To reduce the hustle, they could certainly be initialized by default. The reason for them to be initalized separately is the treeshekability.